### PR TITLE
Instead of depending on make and go to build, lets depend on docker. Bump the simble dependency version.

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

**Comment by EricWittmann** [comment]
All set on this?

**Comment by chirino** [comment]
I think we are all set now, this should go green in circleci.
One of the circleci admin will need to DOCKER_USERNAME and DOCKER_PASSWORD env vars in circleci config settings so that the push to docker hub can work.

**Comment by EricWittmann** [comment]
OK I can do that, no problem.  Does CircleCI only push to docker hub when building master?  (and NOT push when building a pull request)  If so, how does it know?

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,24 +19,61 @@ jobs:
...
         
       # build and run tests
-      - run: mvn clean package
+      - run: sudo apt-get install -y rsync
+      - run: mvn clean package --batch-mode  -P image -Dfabric8.mode=kubernetes
+
+push_images:
+  deploy:
+    command: |
+      if [ "${CIRCLE_PROJECT_USERNAME}" != "Apicurio" ]; then
```
Comment (chirino) [inline .circleci/config.yml]: I think this is the magic

**Pull request merged**